include/RAJA/policy/tensor/arch/avx512/avx512_int64.hpp

```diff
@@ -126,7 +126,12 @@ namespace expt
       RAJA_INLINE
       self_type &load_packed(element_type const *ptr){
 			  // AVX512F
-        m_value = _mm512_loadu_epi64(ptr);
+        #if (defined(__GNUC__) && ((__GNUC__ >= 7) && (__GNUC__ <= 10))) \
+            (!defined(SYCL_LANGUAGE_VERSION) && defined(__INTEL_LLVM_COMPILER))  // Check for oneapi's icpx.
+        m_value = _mm512_maskz_loadu_epi64(~0, ptr);  // May cause slowdown due to looping over 8 bytes, one at a time.
```

Comment:
Should this `~0` be a 64-bit integer?

Reply:
I guess this is ok because the value is sign-extended from 32 to 64 bits, but its still a bit sketchy to me.

Reply:
This should be ok because the instruction is only looking at the first 8 bits of `~0`. https://www.intel.com/content/www/us/en/docs/intrinsics-guide/index.html#text=_mm512_maskz_loadu_epi64&ig_expand=4330,4376,4046,4045,3965,2319,2319,4341&avx512techs=AVX512F

This, and other masked intrinsics, operate in the same way. The mask is specifying which 64-bit chunks to load or store (up to a max of 8 consecutive chunks).